multimedia/fluidsynth/Portfile

```diff
@@ -17,14 +17,13 @@ long_description    FluidSynth is a cross-platform real-time software synthesize
 
 platforms           darwin
 
-homepage            http://fluidsynth.sourceforge.net/
-master_sites        sourceforge:project/${name}/${name}-${version}
-use_bzip2           yes
+homepage            http://www.fluidsynth.org/
 
-checksums           rmd160  27b17e1e097004dd3cf8fb88d8e4499eeb339ea8 \
-                    sha256  d28b47dfbf7f8e426902ae7fa2981d821fbf84f41da9e1b85be933d2d748f601
+checksums           rmd160  dca66848a63d51ab6b26f5d649d7e8cd688514b6 \
+                    sha256  7fa7f07cb2a56480df2f80478d68f452a2748c7583aeff010e89af683075c708 \
+                    size    1373337
 
-depends_build       port:pkgconfig
+depends_build       port:doxygen
```

Comment:
doxygen is required to build the (API?) documentation, but I'm not seeing any indication that this is done in a regular build. Adding `-DCMAKE_DISABLE_FIND_PACKAGE_Doxygen=ON` to the configure arguments disables the doxygen lookup (and calling).

Reply:
I've seen this in your version and what you do makes sense. I noticed that the port was opportunistically using the dependency, but I'm also way more happy to leave it out.